2025/07/08 03:28:00 [error] 11#11: *21 [lua] api_entry.lua:346: handle_request(): Route handler error [GET /log/reports]: /app/lua/core/logs.lua:215: attempt to call field 'get_reports_dir' (a nil value), client: 172.26.0.1, server: _, request: "GET /asd1239axasd/api/log/reports HTTP/1.1", host: "localhost:8765", referrer: "http://localhost:8765/asd1239axasd"
2025/07/08 03:28:02 [error] 7#7: *17 [lua] api_entry.lua:346: handle_request(): Route handler error [GET /log/reports]: /app/lua/core/logs.lua:215: attempt to call field 'get_reports_dir' (a nil value), client: 172.26.0.1, server: _, request: "GET /asd1239axasd/api/log/reports HTTP/1.1", host: "localhost:8765", referrer: "http://localhost:8765/asd1239axasd"
2025/07/08 03:29:44 [error] 7#7: *17 [lua] api_entry.lua:346: handle_request(): Route handler error [GET /log/reports]: /app/lua/core/logs.lua:218: attempt to call field 'check_dir_exists' (a nil value), client: 172.26.0.1, server: _, request: "GET /asd1239axasd/api/log/reports HTTP/1.1", host: "localhost:8765", referrer: "http://localhost:8765/asd1239axasd"
2025/07/08 03:29:50 [error] 9#9: *19 [lua] api_entry.lua:346: handle_request(): Route handler error [GET /log/reports]: /app/lua/core/logs.lua:218: attempt to call field 'check_dir_exists' (a nil value), client: 172.26.0.1, server: _, request: "GET /asd1239axasd/api/log/reports HTTP/1.1", host: "localhost:8765", referrer: "http://localhost:8765/asd1239axasd"
